pkg/detectors/demio/demio.go

```diff
@@ -0,0 +1,82 @@
+package demio
+
+import (
+	"context"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct{}
+
+// Ensure the Scanner satisfies the interface at compile time
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	client = common.SaneHttpClient()
+
+	//Make sure that your group is surrounded in boundry characters such as below to reduce false positives
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"demio"}) + `\b([a-z0-9A-Z]{32})\b`)
+	secretPat = regexp.MustCompile(detectors.PrefixRegex([]string{"demioSecret"}) + `\b([a-z0-9A-Z]{15})\b`)
```

Comment:
suggestion: `demioSecret` is unlikely to be found, recommended to use just `demio`.